signac/__main__.py

```diff
@@ -718,16 +714,12 @@ def main_config_show(args):
     if args.local and args.globalcfg:
         raise ValueError("You can specify either -l/--local or -g/--global, not both.")
     elif args.local:
-        for fn in config.CONFIG_FILENAMES:
-            if os.path.isfile(fn):
-                if cfg is None:
-                    cfg = config.read_config_file(fn)
-                else:
-                    cfg.merge(config.read_config_file(fn))
+        if os.path.isfile(config.PROJECT_CONFIG_FN):
+            cfg = config.read_config_file(config.PROJECT_CONFIG_FN)
     elif args.globalcfg:
-        cfg = config.read_config_file(config.FN_CONFIG)
+        cfg = config.read_config_file(config.USER_CONFIG_FN)
     else:
-        cfg = config.load_config()
+        _, cfg = config.load_config()
```

Comment:
From a directory with no signac project config (and no global config), I execute `signac config verify`.
- On `master` and `next`, I get the output `project : Possibly invalid or missing.`
- On this branch, I get the output `Passed.` That's a regression.

I caught this because I saw the codecov was missing tests for `signac config verify`. Thanks codecov!